media-sound/mpdscribble/mpdscribble-0.22.ebuild

```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2018 Gentoo Foundation
+# Copyright 1999-2019 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=6
```

Comment:
Can we update to EAPI=7 at this point?

Reply:
I think so, looking at the docs I don't see any removed/banned things used in these ebuilds.